inventory/technical/inventory_valuation.rst

```diff
@@ -0,0 +1,95 @@
+===============================
+Inventory Valuation In Detail
+===============================
+
+The following chapter is more technical and explains how specific elements are computed.
+Basically the elements below are computed for a product.product not for a product.template.
+When calculated for different elements, it may differs.
+
+On Hand quantities
+*******************
+
+- For product.template
+
+Sum of the quantities of all the product.product (variants)
+
+- For product.product
+
+TODO:
+
+
+Inventory Valuation (Current Inventory)
+*****************************************
+This is specified in odoo/addons/stock_account/models/product.py mostly in the method _compute_stock_value() (see refA_)
+
+Fifo Manual
+-------------------
+
+TODO:
+
+Fifo Automated
+-------------------
+
+The quantity is the sum of the quantity for stock.move of this product with a specific domain.
+The value is the sum of the remaining_value for stock.move of this product with a specific domain.
```

Comment:
I would give a small explanation of what the remaining_value of a product is, like how it's calculated.

Reply:
The remaining_value is a field of the sock.move holding the remaining value of the product in this location. So each time the valuation changes, the remaining_value is also updated.